### PR TITLE
Use htmltest instead of linkchecker

### LINKED ISSUE #19303: Linkchecker improovement
Hello,
Please, the current linkchecker script that validates current Eclipse Che PRs get stuck on an error caused by a website `denial of service attack` protection, causing a delay in the merging process. When this link test isn't finished successfully, we can merge a PR. 

The problem is, that the test fails on a link that is active and OK, but a "Max retries exceeded" counter of the affected page blocks access to that page (caused by several connection attempts in a short time), and linckcecker threatens it as a broken link.

To work around this issue, we set the `ignorewarnings=url-rate-limited` parameter in linkcheckerrc configuration, which should make the linkchecer validation immune against this error:

```
# Ignore the comma-separated list of warnings.
ignorewarnings=url-rate-limited
```

But it seems it doesn't work as we expected and the script ends with a failure that blocks a merge.

Example
```
URL        `https://github.com/che-incubator/chectl#user-content-chectl-serverdeploy'
Parent URL http://localhost:4000/che-7/administration-guide/configuring-openshift-oauth/, line 725, col 1
Real URL   https://github.com/che-incubator/chectl#user-content-chectl-serverdeploy
Result     Error: ConnectionError: HTTPSConnectionPool(host='github.com', port=443): Max retries exceeded with url: /che-incubator/chectl (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f57cadda6a0>: Failed to establish a new ...
```
Can you help us to tune up the linkchecker config so that our process of adding new documentation could be as fast as we can deliver?
Thank you.

### PR DESCRIPTION
**Pull request by themr0c**
Fixes https://github.com/eclipse/che/issues/19303

linchecker is:
* slow => eats our GH Actions minutes budget
* not reliable on anchors
* has no caching => triggers DOS protection on external servers

htmltest is:
* fast
* use cache to be even faster

See: https://github.com/wjdp/htmltest

DONE: A config file for htmltest .  It works with an almost empty ignore list (only 3 items)!
DONE: Dockerfile to build and embed htmltest . Basic tests on the new container.
DONE: scripts in tools to use htmltest
WIP: GitHub action to use htmltest (I wrote it but I need to test it)
WIP: gulpfile to use htmltest
TODO: Extended tests with the new container
TODO: build the container on quay.io

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

File: tools/podmanlinkchecker.sh [hunk 1]
```diff
@@ -19,4 +19,9 @@ elif command -v docker
 else echo "No installation of podman or docker found in the PATH" ; exit 1
 fi
 
-${RUNNER} exec -ti che-docs "./tools/linkchecker.sh"
+# ${RUNNER} exec -ti che-docs "./tools/linkchecker.sh"
```
Comment (benoitf) [inline tools/podmanlinkchecker.sh]: we don't remove linkchecker tool if we use htmltest instead ?

**Review by benoitf — APPROVED** [review]
overall LGTM

File: tools/podmanlinkchecker.sh (hunk 1 above)
Comment (themr0c) [reply tools/podmanlinkchecker.sh]: Correct, I should rename the script

Comment (themr0c) [reply tools/podmanlinkchecker.sh]: Removed linkchecker scripts

**Review by yhontyk — APPROVED** [review]

**Comment by themr0c** [comment]
First run on workspaces.openshift.com:

Screenshot from 2021-03-25 12-13-34

Subsequent run:

Screenshot from 2021-03-25 12-18-25

**Comment by themr0c** [comment]
And the GH Action:

Screenshot from 2021-03-25 13-28-15

**Pull request merged**